content/en/_index.html

```diff
@@ -36,29 +36,29 @@
   - title: Easy to extend
     icon: fas fa-up-right-and-down-left-from-center
     description: >
-      Each package in the platform consists of a set of YAML files. Therefore, anyone with some familiarity with Kubernetes primitives can modify or expand the platform. Delivery of packages to the system is handled by FluxCD, a well-known and widely used tool in the community.
-  - title: Cost efficiency and high performance
+      Each package in the platform consists of a set of YAML files. Therefore, everyone having some familiarity with Kubernetes primitives can modify or expand the platform. Package delivery is reliably handled by FluxCD, a well-known and widely used tool.
+  - title: Cost-efficient and high-performing
```

Comment:
"Cost-efficient and high-performing" looks like alien words. I don't know why.

Reply:
reverted